ui/components/src/LineChart/LineChart.stories.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react';
+
+import { StoryFn, Meta } from '@storybook/react';
+import { LineChart } from '@perses-dev/components';
+
+// More on default export: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+  title: 'LineChart',
+  component: LineChart,
+  args: {
+    height: 200,
+    data: {
```

Comment:
Doing a quick and dirty example here to get a chart working. Will likely want to extract this into something cleaner in follow-ups.

Reply:
There's def some weirdness with some of the chart stuff that probably needs some adjusting. Something we'll want to sort out in follow-ups as we work more on the embeddability use cases.